RELEASE-NOTES.md

```diff
@@ -23,10 +23,10 @@ if __name__ == "__main__":
             ),
         ),
         entrypoint="flows.py:my_flow",
-    ).deploy(name="my-deployment", work_pool_name="above-ground")
+    ).serve(name="my-deployment")
 ```
 
-Here's an example of loading and deploying a flow from an S3 bucket:
+Here's an example of loading and deploying a flow from a public S3 bucket:
```

Comment:
This doesn't necessarily need to be a public bucket, but the key is that the block has already been saved.

```suggestion
Here's an example of loading and deploying a flow from an S3 bucket:
```